From d5dca0c5588a2fcb79e44e754019b6333bc4530d Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 19 Jan 2024 19:02:20 +0100
Subject: [PATCH 545/841] feat(stm32mp25-fdts): add sink-source for LDO3 with
 DDR4

Add the "st,regulator-sink-source" indication in device tree for
VTT_DDR = LDO3 of DDR4 powered by STPMIC25 as it was done for STPMIC1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git de4d82d2a3598b657d7e50b42cf472736726e38a feat(stm32mp25-fdts): add sink-source for LDO3 with DDR4]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I15e879f1995cab2ee7ee5df911d404ca159cde02
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/355042
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/375637
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 66eac6ff6..0e6e8d3af 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -124,6 +124,7 @@
 			};
 			vtt_ddr: ldo3 {
 				regulator-name = "vtt_ddr";
+				st,regulator-sink-source;
 			};
 			vdd3v3_usb: ldo4 {
 				regulator-name = "vdd3v3_usb";
-- 
2.47.2

